getting-started-publishing/publishing-descriptions.md

```diff
@@ -1,15 +1,55 @@
 ---
 info_en: Page about "Getting started - Publishing descriptions"
-info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger"
+info_no: Side om "Komme i gang - Publisere beskrivelser/kataloger via høsteadmin-grensesnittet"
 description: Skal gi overordnet informasjon om hvordan publisere beskrivelser/kataloger til data.norge.no
-url: https://data.norge.no/getting-started/publishing-data
-note: Vurder å slå sammen med "Creating descriptions"
+url: https://data.norge.no/getting-started/setup-harvesting
 ---
 
-# Publisere beskrivelser
+# Hvordan publisere beskrivelser
 
-## Guide
+Har du brukt registreringsløsningen til å lage beskrivelser, og publisert dem derifra, blir katalogen automatisk lagt til for høsting.
 
-[Beskrive admin-grensesnittet, se https://data.norge.no/publishing/about-harvesting]
+Ønsker du å starte en høsting av beskrivelsene dine manuelt, gjør du det fra [administrasjons-grensesnittet for høsting](https://admin.fellesdatakatalog.digdir.no/data-sources).
+Du må først tilgjengeliggjøre beskrivelsen i RDF-format på nettet, og registrere endepunktet i administrasjonsgrensesnittet. Slik gjør du det:
 
-## ?
+## Registrere høsteendepunkt
+
+**Før du begynner:**
+
+- Høsteendepunktet må være tilgjengelig på nettet (eventuelt sikret med en API-nøkkel).
+- Dataen som returneres må være i et av [formatene data.norge.no støtter]().
```

Comment:
```suggestion
- Dataene som returneres må være i et av [formatene data.norge.no støtter]().
```

Reply:
Legge til nytt punkt om at dataene må være beskrevet i henhold til gjenldende standard for ressurstypen?

Reply:
Ja, kan presisere at det må være i henhold til spec for å vises korrekt på data.norge.no. Strengt tatt trenger det ikke å være i henhold til spec, de feltene som ikke stemmer vises bare ikke.

Reply:
Dette er også beskrevet i creating-descriptions.md, så vet ikke om det er nødvendig å presisere her også.